wpilibc/src/main/native/include/frc/ADIS16448_IMU.h

```diff
@@ -352,6 +362,7 @@ class ADIS16448_IMU : public wpi::Sendable,
   /** @brief Internal Resources **/
   DigitalInput* m_reset_in = nullptr;
   DigitalOutput* m_status_led = nullptr;
+  Rotation3d angleOffset = Rotation3d{0_deg, 0_deg, 0_deg};
```

Comment:
Remove this- It's causing confusion (`Reset()` updates `angleOffset` but the angle getters use `m_angleOffset`), uses a somewhat more computationally expensive constructor, and doesn't match Hungarian notation.